### PR TITLE
feat: Export docs to PDF

### PR DESCRIPTION
**Pull request by Selflocking**
close: #68 
Preview: https://github.com/Selflocking/casbin-website-v2/releases/tag/pdf

ci use [docusaurus-prince-pdf](https://github.com/signcl/docusaurus-prince-pdf) to generate pdf. When successfully built, ci will remove the release tagged pdf (if exists), and create a new release tagged pdf

### CODE REVIEW

File: .github/workflows/pdf.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        
+    - name: Upload Release
+      uses: softprops/action-gh-release@v1
+      with:
+        name: Casbin Docs
+        tag_name: pdf
+        files: pdf/*.pdf
+        body: auto-generated Casbin Docs in PDF format
```
Comment (Selflocking) [inline .github/workflows/pdf.yml:50]: Release body
image

**Comment by hsluoyz** [comment]
@Selflocking looks good! Does it generate for each PR merge?

**Comment by hsluoyz** [comment]
@Selflocking can it be less frequent? Like a release per two weeks?

**Comment by Selflocking** [comment]
> @Selflocking can it be less frequent? Like a release per two weeks?

The old release will be removed when build successfully, so there is only one release in repo

**Comment by hsluoyz** [comment]
@Selflocking this sounds good! Plz also add this to `casdoor-website`

**Pull request merged**